README.md

```diff
@@ -133,6 +140,25 @@ repository’s Security tab. For more information, see the Scorecards GitHub
 Action
 [installation instructions](https://github.com/ossf/scorecard-action#installation).
 
+### Scorecards REST API
+
+To query pre-calculated scores of OSS projects use the REST API
```

Comment:
add comma after "projects": To query pre-calculated scores of OSS projects, ...

Reply:
Done.